File: script/python/game/db/db_redis.py

import PyDb


class DBRedis(object):

    def __init__(self, redis_ip, redis_port):
        self._redis_inst = PyDb.Redis(redis_ip, redis_port)
        assert(self._redis_inst is not None)

    def exec_redis_cmd(self, cmd, *p):
        return self._redis_inst.execRedisCmd(cmd.format(*p))

File: script/lua/game/db/db_redis.lua
require("util.logger")

clsRedis = clsObject:Inherit("clsRedis")

function clsRedis:__init__(redisIp, redisPort)
    self.redis = LuaRedis.new(redisIp, redisPort)
    local ret = self.redis:execRedisCmd("hget test a")
    print("redis return ", type(ret), ret)
end
